kube/services/arborist/arborist-deploy.yaml

```diff
@@ -106,6 +106,15 @@ spec:
               configMapKeyRef:
                 name: manifest-global
                 key: sync_from_dbgap
+          - name: FENCE_PUBLIC_CONFIG
+            valueFrom:
+              configMapKeyRef:
+                name: manifest-fence
+                key: fence-config-public.yaml
+                optional: true
+          - name: YAML_MERGE_SCRIPT
+            value: |
+GEN3_YAML_MERGE   
```

Comment:
I assume it was just easier to include the script this way, rather than writing an actual file during setup-secrets and loading into configmaps/secrets and then mounting it in here?

Reply:
yeah - good call - don't know why I talked myself into that :face_with_head_bandage: